nav2_map_server/README.md

```diff
@@ -63,41 +138,106 @@ free_thresh: 0.196
 ```
 
 The Nav2 software retains the map YAML file format from Nav1, but uses the ROS2 parameter
-mechanism to get the name of the YAML file to use. This effectively introduces a
-level of indirection to get the map yaml filename. For example, for a node named 'map_server',
-the parameter file would look like this:
+mechanism to get the name of the YAML file to use. This effectively introduces a level of indirection to get the map yaml filename. For example, for a node named 'map_server', the parameter file would look like this:
 
-```
-# map_server_params.yaml
+```yaml
+# map_server_params_2d.yaml
 map_server:
     ros__parameters:
-        yaml_filename: "map.yaml"
+        yaml_filename: "map_2d.yaml"
 ```
 
-One can invoke the map service executable directly, passing the params file on the command line,
-like this:
+Below is the process to use `parameters file` with ROS2:
 
+```shell
+# Provide the params filename accordinaly
+# Specify the parmeters file
+############################
+# For 2D Map Server
+$ ros2 run nav2_map_server map_server_2d --ros-args --params-file <map_server_params_2d.yaml>
+# For 3D Map Server
+$ ros2 run nav2_map_server map_server_3d --ros-args --params-file <map_server_params_2d.yaml>
```

Comment:
Misprint in last yaml name: is there should be map_server_params_3d.yaml ?
Also, I think you can remove first three comment lines from "Provide the params...." up to "####...". This has already been described in more detail earlier. There is no need to repeat.